src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql

```diff
@@ -1,4 +1,4 @@
-{% set periods = ["BOY", "BOY->MOY", "MOY", "MOY->EOY", "EOY"] %}
+/*{% set periods = ["BOY", "BOY->MOY", "MOY", "MOY->EOY", "EOY"] %}

```

Comment:
you can delete this commented out part

Reply:
I left it there because this is work in progress and I dont want to lose your code until Im sure I've taken all the best parts of it and we can review it together. I can save it somewhere else, too, I guess?

Reply:
no prob, saw a lot of this code in the updated part. maybe move it to the bottom?